browser/ui/views/split_view/split_view_separator.h

```diff
@@ -61,6 +69,9 @@ class SplitViewSeparator : public views::ResizeArea,
   raw_ptr<SplitViewSeparatorDelegate, DanglingUntriaged> resize_area_delegate_ =
       nullptr;
 
+  raw_ptr<BraveMultiContentsViewSeparatorDelegate> separator_delegate_ =
+      nullptr;
+
```

Comment:
as per https://github.com/brave/brave-core/pull/29154#discussion_r2103675213 I think we should update this to be:

```suggestion
  raw_ptr<views::ResizeAreaDelegate, DanglingUntriaged> resize_area_delegate_ =
      nullptr;

  raw_ptr<SplitViewSeparatorDelegate> separator_delegate_ =
      nullptr;

```